library/Freckle/App/Wai.hs

```diff
@@ -11,7 +11,7 @@ module Freckle.App.Wai
   , denyFrameEmbeddingMiddleware
   ) where
 
-import Freckle.App.Prelude
+import Freckle.App.Prelude hiding (decodeUtf8)
```

Comment:
What's conflicting?

Reply:
We have the following defined below
```haskell
decodeUtf8 :: ByteString -> Text
decodeUtf8 = decodeUtf8With lenientDecode
```

Reply:
Isn't that what `Prelude.decodeUtf8` is? In other word, why hide instead of removing that?

Reply:
Looking at `Data.Text.Encoding` it looks like it's defined as
```haskell
decodeUtf8 = decodeWithUtf8 strictDecode
```

Have I exported the wrong thing?

Reply:
Good question. I didn't realize that and have always operated under the assumption it was `lenientDecode`. Probably a decent argument for not exporting this and requiring explicit local redefinition :upside_down_face: .

If everyone discussing this in Guild knew that but me then sounds like this PR is totally right as-is. If others were confused or didn't think about this, it could be worth re-opening the discussion. :shrug: 

For what it's worth, when I wrote the local `decodeUtf8` here, I did not pick `lenientDecode` on purpose, I was simply attempting to redefine the normal one. So no `hiding` and removing it would change behavior but match original intent.